azure/dns/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "subdomain" {
+  type = string
+}
+
+variable "tags" {
+  type = any
+}
+
+variable "zone_name" {
+  type = string
+}
+
+variable "zone_resource_group_name" {
+  type = string
+}
+
+variable "resource_group_name" {
+  type = string
+}
+
+variable "azurerm_linux_function_app" {
+  type    = any
+  default = null
+}
+
+variable "azurerm_linux_web_app" {
```

Comment:
is this effectively app service?

Reply:
Yes! I'm down for better variable names here. I followed a similiar pattern I used for Cloud Run and Cloud Functions. 

https://github.com/massdriver-cloud/terraform-modules/blob/main/gcp-endpoint/variables.tf#L21-L29

Funny enough, Azure App Service feels like Cloud Functions and Function Apps feel like Cloud Run.